sympy/polys/ring_series.py

```diff
@@ -368,8 +360,8 @@ def rs_subs(p, rules, x, prec):
                 if r == 0 and (i, q) in d:
                     d[(i, power)] = rs_square(d[(i, q)], x, prec)
                 elif (i, power - 1) in d:
-                    d[(i, power)] = rs_mul(d[(i, power - 1)], d[(i, 1)], \
-                        x, prec)
+                    d[(i, power)] = rs_mul(d[(i, power - 1)], d[(i, 1)],
+                                           x, prec)
```

Comment:
No hanging indent here.
I tend to add consistent line breaks, giving me

```
                    d[(i, power)] = rs_mul(
                        d[(i, power - 1)],
                        d[(i, 1)],
                        x,
                        prec)
```

Since the argument list isn't *that* long, this might be an option, too:

```
                    d[(i, power)] = rs_mul(
                        d[(i, power - 1)], d[(i, 1)], x, prec)
```

I prefer breaking at the outermost precedence level, which would be around the assignment operator:

```
                    d[(i, power)] = \
                        rs_mul(d[(i, power - 1)], d[(i, 1)], x, prec)
```

I'm not too confident that this last variant is the best option; the rule is to double-check for readability after formatting changes.